### ISSUE TITLE
the pre-trained model can't achieve 88.38% validation accuracy

### ISSUE THREAD

**Issue by dmortem** [xingyizhou/pytorch-pose-hg-3d#35]
Hi, xingyi,
I use your codes and load the pre-trained model you provided. But I can only achieve 85.789% on the validation set without training, while you said it is 88.38% accuracy. After training for 30+ epochs, the validation accuracy is around 86%. I think the validation set is fixed, no random split, so the accuracy should be the same. Why does that happen?
![default](https://user-images.githubusercontent.com/20540613/44244865-07100900-a1a4-11e8-8b58-c8526a30f9f5.png)
Thanks a lot!
